skopt/tests/test_common.py

```diff
@@ -422,7 +422,7 @@ def test_early_stopping_delta_x(minimizer):
     res = minimizer(bench1,
                     callback=DeltaXStopper(0.1),
                     dimensions=[(-1., 1.)],
-                    x0=[[0.1], [-0.1], [0.9]],
+                    x0=[[-0.1], [0.9], [0.1]],
```

Comment:
This test ended up failing because we get other(?) random numbers and don't have two successive values of `x` be close enough. The first "random" point we draw is at `x=0.01` so by swapping the order of the initial points we hit the stopping criteria right away.